src/board-explorer-app/board-explorer-app.html

```diff
@@ -1185,9 +1183,13 @@
          href += "#" + board;
       }
 
+      if (this.category) {
+        href += "&" + this.category.slug;
```

Comment:
I had to change the separator in the uri due ```/```  it was causing a lot of pain mixing it within the filters.